### ISSUE TITLE
Experiments to improve/understand agents:

### ISSUE THREAD

**Issue by FelipeMartins96** [FelipeMartins96/rl-jax#10]
List of things to try on this repo:
- Profile agents using jax profiler
- Run 0.0.3 on rsoccer_gym VSS-v0 environment
- SAC learnable temperature
- Batch Normalization
- Hindsight experience replay
- Wandb episode statistics wrapper
- Use jax rng everywhere to force reproducibility 

DDPG:
- [x] Change network to Double Q (togglable?)
- [x] Add OU exploration noise (togglable?)
- [ ] Hyperparameter for number of observations per update step and number of gradients per update step

TD3:
- [ ] Implement [TD3](https://arxiv.org/pdf/1802.09477.pdf)
